code/parse/sexp.cpp

```diff
@@ -25590,7 +25590,7 @@ int get_sexp_main()
 	return start_node;
 }
 
-int run_sexp(const char* sexpression)
+int run_sexp(const char* sexpression, bool run_eval_num, bool *check_for_nan)
```

Comment:
`check_for_nan` is a bit misleading here since at least for me that implies that it changes the behavior of the function. Maybe:
```suggestion
int run_sexp(const char* sexpression, bool run_eval_num, bool *is_nan_out)
```

Reply:
Mmm, I changed it to `is_nan_or_nan_forever`.

Reply:
And changed it for real this time.